lib/learn_to_rank/ranker.rb

```diff
@@ -0,0 +1,33 @@
+require "httparty"
+
+module LearnToRank
+  class Ranker
+    # Ranker takes feature sets and requests new scores for them
+    # from a pre-trained model.
+    def initialize(feature_sets = [])
+      @feature_sets = feature_sets
+    end
+
+    def ranks
+      fetch_new_scores(feature_sets).fetch('results', [])
+    end
+
+  private
+
+    attr_reader :feature_sets
+
+    def fetch_new_scores(examples)
+      url = "http://172.17.0.3:8501/v1/models/ltr:regress"
```

Comment:
I don't think it's guaranteed that docker containers will always use the same subnet, so it would be better if the container mapped a port from localhost